checkout/interfaces/controller/checkoutcontroller.go

```diff
@@ -84,6 +84,8 @@ type (
 		redirectToCartOnInvalideCart    bool
 		privacyPolicyRequired           bool
 
+		DevMode bool `inject:"config:debug.mode"`
```

Comment:
Direct injections are deprecated. Please use `CheckoutController`'s `Inject` function and pass an anonymous config struct (like it is done in the `CartViewController` for example)

Reply:
Updated code to use the chainable injection method